### ISSUE TITLE
Problem with entries migration

### ISSUE THREAD

**Issue by AlexandrZherdev** [Firstborn/Craft-CMS-Migration-Manager#8]
When I try to upload entries migration, I get next:

Fatal error: Call to undefined function \Craft\MigrationManagerHelper::getAssetByHandle() in C:\OSPanel\domains\dogshare.prod\craft\plugins\migrationmanager\services\MigrationManager_BaseContentMigrationService.php on line <i>277</i>

I resolved this problem by replacing line 277 with 

$arTmp = explode('::',$func);
$arTmp[0]::$arTmp[1]($el ement);
$item = $arTmp[0]::$arTmp[1]($el ement);

in MigrationManager_BaseContentMigrationService class

**Comment by AlexandrZherdev** [comment]
Hi.
I found one more reason of the problem. Problem is related with creating JSON for entries. There is not correct writing of checboxes and radiobuttons.

Craft saves entry with method setContentFromPost, but it don't save in format  [label=>, value=>, selected=>] for radibuton and checkbox

For radiobutton we tried to save only value (for example dogGender=>male) and it worked.

For checkbox we tried to save array of values(for example whatLookingFor => [value1,value2,value3]) and it worked.

Look at "dogSituation", "dogGender", "whatLookingFor"
It is yours:
![your](https://user-images.githubusercontent.com/32765290/31987458-8f971016-b974-11e7-8aee-c2e4f3674000.png)
And ours:
![our](https://user-images.githubusercontent.com/32765290/31987477-9c39587e-b974-11e7-8319-542163ca7ce0.png)
Maybe this situation is specific only for our case.
